### DISCUSSION TITLE
rename no_auto

### DISCUSSION THREAD

**Discussion by bessarabov** [umputun/spot#52] [General]
The fragment from README.md:

```
Each command type supports the following options:

ignore_errors: if set to true the command will not fail the task in case of an error.
no_auto: if set to true the command will not be executed automatically, but can be executed manually using the --only flag.
```

What do you think of renaming `no_auto` to `only_manual`? This plays well with the existing cli option `--only`

**Comment by umputun** [comment]
I don't like `no_auto` too much, but `only_manual` is not that better. maybe just `manual` not sure. For me `no_auto` seems to be the better option from these three

**Comment by bessarabov** [comment]
Ok. Thank you. I'm closing this discussion.
